modules/mod_translation/controllers/controller_language_set.erl

```diff
@@ -56,6 +57,7 @@ moved_permanently(Context) ->
     {false, Context}.
 
 
+-spec add_language(string(), #context{}) -> binary().
```

Comment:
The spec in line 60 uses a record, please define a type for the record and use that instead.